.github/workflows/maven.yml

```diff
@@ -15,16 +15,20 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Install modules
-      run: npm i
-      working-directory: ui
-    - name: Run ESLint
-      run: npm run lint
-      working-directory: ui
-    - name: Set up JDK 11
-      uses: actions/setup-java@v1
-      with:
-        java-version: 11
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      - uses: actions/checkout@v2
+      - name: Install modules
+        run: npm i
+        working-directory: ui
+      - name: Run ui Tests
+        run: npm run test
```

Comment:
Are you sure this will work on GitHub, inside the Docker container for running the steps?